lib/network_monitor.c

```diff
@@ -1283,6 +1500,8 @@ bool mptcpd_nm_register_ops(struct mptcpd_nm *nm,
         if (nm == NULL || ops == NULL)
                 return false;
 
+        test_net_v4.s_addr = htonl(0xc0000201);
```

Comment:
Maybe add comment to explain what this specific IPv4 address is?
```suggestion
        // See IETF RFC 5737: IPv4 Address Blocks Reserved for Documentation.
        test_net_v4.s_addr = htonl(0xc0000201);  // 192.0.2.1
```